tests/schema/test.firefox-schemas-import.js

```diff
@@ -1171,17 +1162,14 @@ describe('firefox schema import', () => {
         .withArgs('https://hg.mozilla.org/mozilla-central/archive/FIREFOX_AURORA_54_BASE.tar.gz')
         .returns(mockStream);
       expect(fs.readdirSync(outputPath)).toEqual([]);
-      return fetchSchemas({ version: 54, outputPath })
-        .then(() => {
-          expect(true).toBeFalsy();
-        }, () => {
-          // Manually remove the tar file since it doesn't get cleaned up.
-          fs.unlinkSync('tmp/FIREFOX_AURORA_54_BASE.tar.gz');
-          expect(true).toBeTruthy();
-        });
+      await expect(
+        fetchSchemas({ version: 54, outputPath })
+      ).rejects.toThrow();
```

Comment:
Same here (if we opt to make the assertions of the previous testcase to also check the error message).